src/styles/components/_contacts.scss

```diff
@@ -0,0 +1,94 @@
+.contacts {
+  padding: 0 20px 100px;
+
+  @media (min-width: $tablet-size) {
+    padding: 0 72px 156px;
+  }
+  @media (min-width: $wide-screen-size) {
+    padding: 0 150px 120px;
+  }
+
+  &__header {
+    margin-bottom: 48px;
+  }
+  &__block {
+    @media (min-width: $tablet-size) {
+      display: flex;
+      gap: 75px;
+    }
+  }
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+    gap: 48px;
+    margin-bottom: 48px;
+    min-width: 237px;
+
+    @media (min-width: $wide-screen-size) {
+      flex-direction: row;
+      gap: 24px;
+    }
+  }
+
+  &__contact-name {
+    margin-bottom: 16px;
+    font-weight: 700;
+  }
+
+  &__contact-type {
+    margin-bottom: 4px;
+    font-weight: 400;
+  }
+
+  &__contact-phone,
+  &__contact-address {
+    font-weight: 700;
+    color: var(--main-text);
+  }
+
+  &__contact-phone {
+    margin-bottom: 8px;
+    text-decoration: none;
+  }
+
+  &__form {
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+    width: 100%;
+
+    &-field {
+      background-color: var(--form-background);
+      border: none;
+      border-radius: 0;
+      height: 48px;
+      padding: 10px;
+
+      &::placeholder {
+        color: var(--form-label-grey);
+        font-family: Cinzel, Georgia, Times, "Times New Roman", serif;
+        line-height: 20px; /* 142.857% */
+      }
+
+      &--message {
+        height: 144px;
+        margin-bottom: 16px;
+      }
+    }
+
+    &-button,
+    &-button:active,
+    &-button:hover {
+      height: 56px;
+      background-color: var(--main-accent);
+      color: var(--main-white);
+      border: none;
+      text-align: center;
+      font-family: inherit;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 27px; /* 168.75% */
```

Comment:
your button without any hover and even without cursor pointer, we can't know that the button was pressed

Reply:
Fixed that